README.md

```diff
@@ -1 +1,338 @@
 ![travis ci](https://travis-ci.org/electrode-io/react-native-electrode-bridge.svg?branch=master)
+
+
+Electrode React Native Bridge
+-----------------------------
+
+This project is essentially a react native library, consisting of a JavaScript module and an associated Android Native Module and iOS Native module.
```

Comment:
essentially - not necessary

Reply:
This project is essentially a react native library, consisting of a JavaScript module and an associated Android Native Module and **an** iOS Native module.